docs/reference.html

```diff
@@ -6,7 +6,7 @@
 
 <h2>Leaflet API reference</h2>
 
-<p>This reference reflects <strong>Leaflet v1.9.2</strong>. Check <a href='reference-versions.html'>this list</a> if you are using a different version of Leaflet.</p>
+<p>This reference reflects <strong>Leaflet </strong>. Check <a href='reference-versions.html'>this list</a> if you are using a different version of Leaflet.</p>
```

Comment:
@Falke-Design A bit late with the review, sorry. This should've been `<strong>Leaflet v1.9.3</strong>`?

Reply:
Good catch!
Why is this not generated automatically? ...

Reply:
Good question, it should have been. I do recall doing all of this manually. We should really find a way to automate the entire release process.